conf/bk_server.conf

```diff
@@ -1080,14 +1089,16 @@ dlog.bkcAckQuorumSize=2
 ### Storage ###
 
 # local storage directories for storing table ranges data (e.g. rocksdb sst files)
-storage.range.store.dirs=data/bookkeeper/ranges
+storage.range.store.dirs=/tmp/bk-ranges
```

Comment:
Is this needed for this PR? Would you mind keeping it as original?

Reply:
> Is this needed for this PR? Would you mind keeping it as original?

well, can keeping it as original.